python_modules/dagster/dagster/_core/secrets/loader.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+from typing import Dict, Optional
+
+from dagster._core.instance import MayHaveInstanceWeakref
+
+
+class SecretsLoader(ABC, MayHaveInstanceWeakref):
+    @abstractmethod
+    def load_env_vars(self, location_name: Optional[str]) -> Dict[str, str]:
```

Comment:
is it worth doing a `context` argument here to set-up for future desired information?